packages/currency/src/erc20/index.ts

```diff
@@ -98,7 +61,7 @@ export function getSupportedERC20Tokens(): ERC20TokenDetails[] {
           network: networkName,
           decimals: token.decimals,
           name: token.name,
-          symbol: `${token.symbol}${networkName !== 'mainnet' ? `-${networkName}` : ''}`,
+          symbol: token.symbol,
```

Comment:
the place where it matters to have the format `symbol-network` is the request's `currency` field, where it uses the `toString()` method; this method still appends the network.